manual/wixtoolset/tools/chocolateyInstall.ps1

```diff
@@ -1,7 +1,13 @@
-﻿$packageName = 'wixtoolset'
-$installerType = 'EXE'
-$32BitUrl = 'http://wixtoolset.org/downloads/v3.10.3.3007/wix310.exe'
-$silentArgs = '/q'
-$validExitCodes = @(0)
+﻿$packageParameters = @{
+  PackageName = 'wixtoolset'
+  SilentArgs = '/q'
+  Url = 'http://wixtoolset.org/downloads/v3.11.0.1507/wix311.exe'
+  Url64bit = ''
+  ValidExitCodes = @('0')
```

Comment:
I tend to use the splat syntax instead of free variables (this is a cut-n-paste job). Its also how the docs show to do it: https://github.com/chocolatey/choco/wiki/HelpersInstallChocolateyPackage